<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsNotifications</class>
 <widget class="QWidget" name="SettingsNotifications">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>357</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="m_checkEnableNotifications">
     <property name="text">
      <string>Enable notifications</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="NotificationsEditor" name="m_editor" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="HelpSpoiler" name="m_lblInfo" native="true"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="m_gbNotificationsType">
     <property name="title">
      <string>Balloon notifications type</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="m_rbNativeNotifications">
        <property name="text">
         <string>Native notifications (tray icon must be enabled)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="m_rbCustomNotifications">
        <property name="text">
         <string>Custom notifications</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QWidget" name="m_wdgCustomNotifications" native="true">
        <layout class="QFormLayout" name="formLayout_3">
         <property name="leftMargin">
          <number>30</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Position</string>
           </property>
           <property name="buddy">
            <cstring>m_cbCustomNotificationsPosition</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="m_cbCustomNotificationsPosition"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Width</string>
           </property>
           <property name="buddy">
            <cstring>m_sbWidth</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="m_sbWidth">
           <property name="suffix">
            <string notr="true"> px</string>
           </property>
           <property name="minimum">
            <number>50</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Margins</string>
           </property>
           <property name="buddy">
            <cstring>m_sbMargin</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="m_sbMargin">
           <property name="suffix">
            <string notr="true"> px</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Screen</string>
           </property>
           <property name="buddy">
            <cstring>m_sbScreen</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="m_sbScreen">
           <property name="value">
            <number>99</number>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="m_lblScreenInfo">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="m_sbOpacity">
           <property name="suffix">
            <string notr="true"> %</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Opacity</string>
           </property>
           <property name="buddy">
            <cstring>m_sbOpacity</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HelpSpoiler</class>
   <extends>QWidget</extends>
   <header>helpspoiler.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NotificationsEditor</class>
   <extends>QWidget</extends>
   <header>notificationseditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_checkEnableNotifications</tabstop>
  <tabstop>m_rbNativeNotifications</tabstop>
  <tabstop>m_rbCustomNotifications</tabstop>
  <tabstop>m_cbCustomNotificationsPosition</tabstop>
  <tabstop>m_sbWidth</tabstop>
  <tabstop>m_sbMargin</tabstop>
  <tabstop>m_sbScreen</tabstop>
  <tabstop>m_sbOpacity</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_checkEnableNotifications</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_editor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_checkEnableNotifications</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_gbNotificationsType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_rbCustomNotifications</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_wdgCustomNotifications</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
